nifi-commons/nifi-properties/src/main/java/org/apache/nifi/util/NiFiProperties.java

```diff
@@ -1532,4 +1713,21 @@ public void validate() {
         }
         // Other properties to validate...
     }
+	
+	//External load balancer, static key
+	public static final String IS_TOKEN_STATIC = "nifi.token.key.static";
+	public static final String TOKEN_KEY_CONFIG = "nifi.token.key.config";
+	/**
+     * This method is used to validate the NiFi properties when the file is loaded
+     * for the first time. The objective is to stop NiFi startup in case a property
+     * is not correctly configured and could cause issues afterwards.
+     */
+    public void validate() {
```

Comment:
This fails the build as there is already a `public void validate()` method in this class

Reply:
Fixed the issue.